.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,17 +6,21 @@ Closes <!--- Insert Issue Number(s) this PR addresses. Start by typing # will op
 
 ### All Submissions:
 
-* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
 * [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
 * [ ] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.
 
+### Contribution Checklist
+
+* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
+* [ ] Have you associated this PR with a `type:` label?
+* [ ] Have you associated a milestone with this PR?
```

Comment:
I'm concerned that external contributors won't know what to do here. In the case of external contributions, it's up to us to target a milestone.

I'd also love to automate this if possible? Because the associated issues also have a milestone this is redundant information that is _guaranteed_ to get out of sync.

Reply:
Yes, I don't recommend this either.

Reply:
It's up to the release manager to manage this